File: translations/codenet/uncommented/C2P/codellama/Code/s647939399.py
#include<stdio.h>
int main():
  char m
  int a,b,c[256],i,j
  for i in range(0,256):
    scanf("%d %c %d",&a,&m,&b)
    if m=='?':
      break
    if m=='+':
      c[i]=a+b
    if m=='-':
      c[i]=a-b
    if m=='*':
      c[i]=a*b
    if m=='/':
      c[i]=a/b
  for j in range(0,i):
    printf("%d\n",c[j])
  return(0)

File: translations/codenet/uncommented/C2Go/deepseek/Code/s647939399.go
package main

import (
	"fmt"
)

func main() {
	var a, b int
	var m byte
	var c []int
	for {
		fmt.Scanf("%d %c %d", &a, &m, &b)
		if m == '?' {
			break
		}
		switch m {
		case '+':
			c = append(c, a+b)
		case '-':
			c = append(c, a-b)
		case '*':
			c = append(c, a*b)
		case '/':
			c = append(c, a/b)
		}
	}
	for _, v := range c {
		fmt.Println(v)
	}
}

